doc/modules/svm.rst

```diff
@@ -623,6 +622,53 @@ bound of the fraction of support vectors.
 It can be shown that the :math:`\nu`-SVC formulation is a reparametrization
 of the :math:`C`-SVC and therefore mathematically equivalent.
 
+
+SVR
+---
+
+Given training vectors :math:`x_i \in \mathbb{R}^p`, i=1,..., n, and a
+vector :math:`y \in \mathbb{R}^n` :math:`\varepsilon`-SVR solves the following primal problem:
+
+
+.. math::
+
+    \min_ {w, b, \zeta, \zeta^*} \frac{1}{2} w^T w + C \sum_{i=1}^{n} (\zeta_i + \zeta_i^*)
+
+
+
+    \textrm {subject to } & y_i - w^T \phi (x_i) - b \leq \varepsilon + \zeta_i,\\
+                          & w^T \phi (x_i) + b - y_i \leq \varepsilon + \zeta_i^*,\\
+                          & \zeta_i, \zeta_i^* \geq 0, i=1, ..., n
+
+Its dual is
+
+.. math::
+
+   \min_{\alpha, \alpha^*} \frac{1}{2} (\alpha - \alpha^*)^T Q (\alpha - \alpha^*) + \varepsilon e^T (\alpha + \alpha^*) - y^T (\alpha - \alpha^*)
+
+
+   \textrm {subject to } & e^T (\alpha - \alpha^*) = 0\\
+   & 0 \leq \alpha_i, \alpha_i^* \leq C, i=1, ..., n
+
+where :math:`e`, :math:`C > 0` and :math:`Q` are the same as in the case of SVC.
```

Comment:
Ok, then, I think, we should also fix

 >  `Q` is an `n` by `n` positive semidefinite matrix, `Q_{ij} \equiv K(x_i, x_j)` and `\phi (x_i)^T \phi (x)` is the kernel

because the kernel is a function of 2 arguments, and relation between `Q` and `\phi` is not clear.